docs/commands/super.md

```diff
@@ -52,27 +52,31 @@ do not haphazardly change when input data changes in subtle ways.
 
 Each `input` argument to `super` must be a file path, an HTTP or HTTPS URL,
 an S3 URL, or standard input specified with `-`.
-These input arguments are treated as if a SQL "from" operator precedes
+These input arguments are treated as if a SQL `FROM` operator precedes
 the provided query, e.g.,
 ```
-super -c "from example.json | select typeof(this)"
+super -c "FROM example.json | SELECT typeof(this)"
 ```
 is equivalent to
 ```
-super -c "select typeof(this)" example.json
+super -c "SELECT typeof(this)" example.json
+```
+and both are equivalent to the classic SQL
+```
+super -c "SELECT typeof(this) FROM example.json"
```

Comment:
I'm still getting the feel for when and how much to reassure readers that standard SQL is still in there. I know we've talked about covering this topic in more detail when we get to overhauling the Language docs. Regardless, my instincts are to make some nods to classic SQL early in individual docs pages since we can't always be sure where users will first land and we can't be confident they'll always click on links when we hope they will. While we hope that users will quickly catch on to SuperSQL's pipe extensions so they can do things with super-structured data that classic SQL can't, I also want to make it as clear as possible to the new users that learning new elements are not a _prerequisite_ to getting started, hence my addition of this new text.

Also related to this is when to use ALL CAPS. Once again, it feels worthwhile to use all caps in contexts where we're showing pure SQL that could drop directly into some other classic relational system. Likewise if we're showing an example that learns mostly/exclusively on SuperSQL shortcuts it seems reasonable to opt for lowercase, even if the example shares a few keywords common in classic SQL. Then there seems to be a gray area when it's a mix. On the whole though, my instincts are to start from all caps when possible to maximize approachability to the SQL-centric audience.